src/styles/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  align-items: flex-end;
+
+  list-style: none;
+  padding: 0;
+
+  width: 166px;
+  height: 16px;
+  margin: 16px 16px 23px;
+
+  &__item {
+    background: url(../images/star.svg) var(--background-properties);
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+
+    &:nth-child(-n + 4) {
+      background: url(../images/star-active.svg) var(--background-properties);
+    }
```

Comment:
Still not fixed, the number of active stars should depend on the modifier. If it's `stars--4` it should 4 active stars, if `stars--3` 3 active stars and so on